package errorHandling;

public class Try_Catch_Finally {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		try {
			System.out.println("Try");
		}
		catch(Exception e) {
			System.out.println(e);
		}
	}

}
